contrib/systemd/prometheus-mumble-exporter.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Prometheus blackbox exporter for Mumble metrics
+After=network.target nss-lookup.target
+
+[Service]
+Type=simple
+EnvironmentFile=-/etc/conf.d/prometheus-mumble-exporter
```

Comment:
Which environment variables does this exporter accept?

Reply:
missing file added in new commit [96be290](https://github.com/mguentner/mumble_exporter/pull/2/commits/96be290a97bfee3b64dbfd8bb98367f609a48af6)